features/externalclustermap.js

```diff
@@ -19,6 +19,10 @@ function openClusterMap(event) {
 			url = "https://codamhero.dev/v2/clusters.php";
 			highlightAfter = true;
 			break;
+		case "Paris":
+			const headerLoginName = document.querySelector("span.login[data-login]")?.textContent;
```

Comment:
Can you change this to `getProfileUserName()`, which is defined in generic/useful.js?